doc/api/util.md

```diff
@@ -71,6 +71,25 @@ callbackFunction((err, ret) => {
 });
 ```
 
+## `util.colorText(format, text)`
+
+<!-- YAML
+added: v18.3.0
+-->
+
+* `format` {string} `format` one of the color format from `util.inspect.colors`
```

Comment:
We should probably link `util.inspect.colors` to https://nodejs.org/api/util.html#customizing-utilinspect-colors.